pkg/k8sutils/k8sutils.go

```diff
@@ -49,3 +50,26 @@ func GetVolumeNamesFromLabelSelector(namespace string, labels map[string]string)
 
 	return volNames, nil
 }
+
+// GetDriverTypeFromPV returns the name of the provisioner driver managing
+// the persistent volume. Supports in-tree and CSI PVs
+func GetDriverTypeFromPV(pv *v1.PersistentVolume) (string, error) {
```

Comment:
Why put this here? Doesn't look like anything but the portworx driver will require this

Reply:
Also, this only really returns driver types that support snapshots (if not CSI), so the name isn't really accurate.

Reply:
Gotcha, good point.